addons/mail/static/src/components/welcome_view/welcome_view.scss

```diff
@@ -1,3 +1,7 @@
+.o_WelcomeView {
+    min-height: 100%;
```

Comment:
What's the change between using height and min-height in this case? If it's because parent is flex, this should have flex-grow-1 instead.